verify.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'enumerable'
+
+class List
+  @list = []
+  include Enumberable
+
+  def initialize(list)
+    @list = list
```

Comment:
- [ ] Hello Team, Good job on your work so far. However, It looks like the changes requested by the previous reviewer to allow the initialize method to accept more list arguments is not added. Normally it is implemented by adding an asterisk before the argument name like this 
```ruby
def initialize(*list)
 list.each { |li| @list << li }
```
Please implement it 🙏